entrypoint_alpine.sh

```diff
@@ -10,6 +10,14 @@ fi
 set -euo pipefail
 IFS=$'\n\t'
 
+# Install Java unlimited strength cryptography
+if [ "${CRYPTO_POLICY}" = "unlimited" ] && [ ! -d "${JAVA_HOME}/jre/lib/security/policy/unlimited" ]; then
+echo "Installing OpenJDK unlimited strength cryptography policy..."
+  apk update
+  apk fix --no-cache openjdk8-jre-lib
+  rm -rf /var/cache/apk/*
```

Comment:
This is also not needed if you use `--no-cache`

Reply:
It is there to remove the package index from `/var/cache/apk`.